src/main/scala/mesosphere/marathon/event/http/HttpEventActor.scala

```diff
@@ -45,10 +44,9 @@ class HttpEventActor(val subscribersKeeper: ActorRef) extends Actor with ActorLo
     response.onComplete {
       case Success(res) =>
         if (res.status.isFailure)
-          log.warning(s"Failed to post $event to $urlString")
-
+          log.warning(s"Post {} to {} resulted in error response {}", event, urlString, res.status)
```

Comment:
I suppose your point was to replace scala string interpolation with the log.warning varargs approach – you missed removing the string interpolation prefix that will lead to a call to `StringContext.s()``